src/predicate/isBrowser.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Checks if the current environment is a browser.
+ *
+ * This function checks for the existence of the `window.document` property,
+ * which only exists in browser environments.
+ *
+ * @returns {boolean} `true` if the current environment is a browser, otherwise `false`.
+ *
+ * @example
+ * if (isBrowser()) {
+ *   console.log("This is running in a browser");
+ *   document.getElementById('app').innerHTML = 'Hello World';
+ * }
+ */
+export function isBrowser(): boolean {
+  return globalThis.window?.document != null;
```

Comment:
what if it is in worker or service worker, will this still work?
that is still browser and not node environment